### ISSUE TITLE
Can't close the popup window

### ISSUE THREAD

**Issue by shyrikoff** [blak3r/yaai#194]
Hi, 

Can anybody help me with nougty popup window?

I have sugarcrm 6.5.16 & callinize community 3.6.9.zip & asterisk 1.8

The problem is to close the popup window after outgoing call.
after click on memo or X button, it close, the correspondent string appers in calls history.
But after F5 button, it again apperas... Can smbdy tell me what to do with this feature?

-browsers: chrome (win7), firefox (ubuntu)
-after F5 or, chnge the link the next appears in browser console 

Refreshing single match callPopups.js:693
Resource interpreted as Script but transferred with MIME type text/plain: "http://192.168.100.37/sugarcrm/custom/modules/Asterisk/include/template/call-template.tmpl". sugar_grp1.js?v=96IXsnSECE1BvPbiVSyp7Q:443
Object {bean_module: "contacts", bean_id: "965ed76f-f418-e5fd-8d32-53a99d5b0cdc", bean_name: "testman testman", bean_description: null, bean_link: "http://192.168.100.37/sugarcrm/index.php?module=Co…lView&record=965ed76f-f418-e5fd-8d32-53a99d5b0cdc"…}

In asteriskLogger during the call next notices:

FOUND outbound CALL
Looking for user extension number in: SIP/alex-00000000
Channel Matched SIP/### style regex. Ext is:alex
### +++ findUserByAsteriskExtension(alex)

! Assigned user id set to 1
 [1403625243.1] Measured call duration is 0 seconds
Bean Id already set by callListener to: 965ed76f-f418-e5fd-8d32-53a99d5b0cdc
PHP Notice:  Undefined index: id in /var/www/sugarcrm/custom/modules/Asterisk/asteriskLogger.php on line 817
! Call is not related to any record (no matches)
! Invalid Arguments passed to setRelationshipBetweenCallAndBean callRecordId=, beanId=965ed76f-f418-e5fd-8d32-53a99d5b0cdc, beanType=Contacts
 Call start was 2014-06-24 15:54:03
PHP Notice:  Undefined index: id in /var/www/sugarcrm/custom/modules/Asterisk/asteriskLogger.php on line 833
 [1403625243.1](OUTBOUND) Now updating record in /Calls/ id=...
PHP Notice:  Undefined index: name in /var/www/sugarcrm/custom/modules/Asterisk/asteriskLogger.php on line 836
NAME:
PHP Notice:  Undefined index: description in /var/www/sugarcrm/custom/modules/Asterisk/asteriskLogger.php on line 837
DESCRIPTION:
PHP Notice:  Undefined index: id in /var/www/sugarcrm/custom/modules/Asterisk/asteriskLogger.php on line 845
PHP Notice:  Undefined index: id in /var/www/sugarcrm/custom/modules/Asterisk/asteriskLogger.php on line 873

During F5 asterisklogger is 
[2014-06-24 16:14:33] Patiently Waiting...! (mem_used: 7341648)

**Comment by ViktorNova** [comment]
I had this issue in the past, I believe it was resolved by giving the correct permissions to the Asterisk manager that yaai uses, from the Asterisk API section in FreePBX

**Comment by blak3r** [comment]
This is a bug I've seen... if numbers get put into asterisk log without
certain fields set... like a click to dial call to a number which asterisk
doesn't actually call this can happen,
ᐧ

On Fri, Jul 25, 2014 at 4:57 PM, Viktor Nova notifications@github.com
wrote:

> I had this issue in the past, I believe it was resolved by giving the
> correct permissions to the Asterisk manager that yaai uses, from the
> Asterisk API section in FreePBX
> 
> —
> Reply to this email directly or view it on GitHub
> http://r.clb.pt/r/f9639494cd483d2ab6b123ce9ae870ee?d=https%3A%2F%2Fgithub.com%2Fblak3r%2Fyaai%2Fissues%2F194%23issuecomment-50203456
> .

## 

Blake Robertson
blake@callinize.com
Callinize

**Comment by dmonies327** [comment]
I know this is old, but I'm having the same issue.

It's as blak3r said, the call was never made. Pressed click to dial, then rejected the call instead of picking it up to trigger asterisk to dial out.

How do I get this popup to go away?

EDIT: Looks like removing it from the asterisk_log table clears it.

**Comment by blak3r** [comment]
yeah, it's with funky calls... look at the asterisk_log database and it's
semi obvious which calls are causing the issue... a simple workaround is to
just update the purge code to delete those.  The root cause is more
complex.  I kinda forget.  Hope I am pointing someone in the right
direction.

I experienced it with click to dial numbers with an extension in it.

I've since been using the professional edition and it's a different code
base altogether.
